### PR TITLE
Add keycloak_realm_rolemapping module to map realm roles to groups

### PR DESCRIPTION
**Pull request by agross**
##### SUMMARY

For my Keycloak setup I need to be able to map realm roles into groups. Until now this collection does not support this.

##### ISSUE TYPE
- New Module/Plugin Pull Request

##### COMPONENT NAME
`keycloak_realm_rolemapping`

##### ADDITIONAL INFORMATION

This is a continuation of #5384 which unfortunately was not merged.

### CODE REVIEW

**Comment by agross** [comment]
Hello @felixfontein,

Thank you for your feedback. I'll continue working on the issues mentioned. I just wanted to get this out the door before people start duplicating work. I was aware this PR isn't perfect yet.

**Comment by felixfontein** [comment]
The `third-party check` check fails from time to time, you can ignore it. (I would rather remove it, but it comes as part of the "collection releasing is managed by Zuul" package, so I'm afraid we cannot remove it so easily...)

File: plugins/modules/keycloak_realm_rolemapping.py [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_realm_rolemapping
+
+short_description: Allows administration of Keycloak realm role mappings into groups with the Keycloak API
+
+version_added: 8.3.0
```
Comment (felixfontein) [inline plugins/modules/keycloak_realm_rolemapping.py:17]:
```suggestion
version_added: 8.2.0
```

File: plugins/modules/keycloak_realm_rolemapping.py [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+        default: 'present'
+        type: str
+        choices:
+            - present
+            - absent
+
+    realm:
+        type: str
+        description:
+            - They Keycloak realm under which this role_representation resides.
```
Comment (felixfontein) [inline plugins/modules/keycloak_realm_rolemapping.py:59]: Here and in a few more places role_representation shows up. Should that be realm_rolemapping?

Comment (agross) [reply plugins/modules/keycloak_realm_rolemapping.py:59]: I don't think so. This module is about mapping a realm's roles (i.e. their role_representation) to a group. The wording is taken from [`keycloak_client_rolemapping`](https://docs.ansible.com/ansible/latest/collections/community/general/keycloak_client_rolemapping_module.html#parameter-realm)

**Comment by felixfontein** [comment]
If nobody objects, I'll merge this in ~a week.

**Comment by felixfontein** [comment]
@agross thanks for your contribution!

**Pull request merged**